posthog/api/cohort.py

```diff
@@ -128,6 +128,20 @@ def _calculate_static_by_csv(self, file, cohort: Cohort) -> None:
     def validate_filters(self, request_filters: Dict):
 
         if isinstance(request_filters, dict) and "properties" in request_filters:
+            if self.context["request"].method == "PATCH":
+                parsed_filter = Filter(data=request_filters)
+                cohort_id = self.instance.pk
+                flags: QuerySet[FeatureFlag] = FeatureFlag.objects.filter(
+                    team_id=self.context["team_id"], active=True, deleted=False
```

Comment:
do we want to search for inactive flags as well?

oh, just tested it out, not a problem, when re-enabling, it will validate the flag properly.